.travis.yml

```diff
@@ -28,11 +24,6 @@ env:
 
 sudo: false
 
-before_cache:
-  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
-
 cache:
   directories:
     - $HOME/.gradle
-    - $HOME/.gradle/caches/
```

Comment:
IMO, we should do the inverse here and remove `$HOME/.gradle`

How are you validating the cache changes?